### PR TITLE
[Storage] Fixed issue that table cmdlet fail with context get from StorageAccount object

### PR DESCRIPTION
**Pull request by blueww**
## Description

This is to fix ICM 539767485.
Hope we can have a hotfix release for it.

I have manually tested it, and will update live test to cover it later.

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Updated Storage account cmdlet output context based on OAuth token
```
Comment (blueww) [inline src/Storage/Storage.Management/ChangeLog.md:21]: This change is in last Sep release, the change log is in wrong place, so move it to the right place.

Comment (msJinLei) [reply src/Storage/Storage.Management/ChangeLog.md:21]: The change isn't included in the previous release https://github.com/Azure/azure-powershell/pull/25954. The change log should be here

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please revise the change log.

File: src/Storage/Storage.Management/ChangeLog.md (hunk 1 above)
Comment (blueww) [reply src/Storage/Storage.Management/ChangeLog.md:21]: fixed.

**Comment by EmmaZhu** [comment]
LGTM

**Pull request merged**